src/components/Input.js

```diff
@@ -0,0 +1,29 @@
+import styled from 'styled-components';
+
+export default function Input({
+  labelText,
+  placeholder,
+  name,
+  value,
+  onChange,
+  required,
+}) {
+  return (
+    <>
+      <label htmlFor={name}>{labelText}</label>
+      <InputStyled
+        id={name}
```

Comment:
I have thought about that in the past a lot, creating a local uuid and assigning it to htmlFor and id will prevent someone from creating 2 inputs with the same id 🤔

Reply:
Hm maybe, but also, mostly the names are different, no?